### PR TITLE
Fixed one broken link issue #9951

### LINKED ISSUE #9951: [Docs] Broken Links
### Current State
If u visit this link (https://docs.meshery.io/guides/configuration-management/working-with-designs) you see almost all the links are redirecting to 404 error page.

### Desired State
It should redirect to the correct url.

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by neerajmanivarnan**
**Notes for Reviewers**
This PR fixes only one broken link .A PR has already submitted  from another branch.

This PR fixes #9951 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit ae2a2b8f01cd64c304dca1c7acb667b14cc08362 at: https://65a2e0aa403346f3401bee7f--meshery-docs-preview.netlify.app

File: docs/pages/guides/configuration-management/configuration-management.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ Meshery's core construct of configuration management is that of a design. A desi
 ## Creating a Meshery Design
 
 You have your choice of how are you would like to create an infrastructure design. You can use the design configurator inside of measure, UI or extensions, like MeshMap, or you can hand code your design, using the mesh model spec, if you like.
- 
-Use Meshery UI or mesheryctl to manage [designs](../tasks/patterns.md).
+
+Use Meshery UI or mesheryctl to manage [designs](/docs/pages/tasks/patterns.md).
```
Comment (leecalcote) [inline docs/pages/guides/configuration-management/configuration-management.md:25]:
```suggestion
Use Meshery UI or mesheryctl to manage [designs]({{site.baseurl}}/tasks/patterns.md).
```

**Review by leecalcote — APPROVED** [review]
Thanks, @neerajmanivarnan! 👍

**Comment by leecalcote** [comment]
Do sign-off your commit, please. See instructions above. 👍

**Comment by l5io** [comment]
🚀 Preview for commit 4522fed44298a88990adc5c478dbdddc4a0b90d2 at: https://65a30dde1e60c10e20297a0c--meshery-docs-preview.netlify.app

**Comment by saurabh100ni** [comment]
Hi @neerajmanivarnan   ,
Thanks for your PR! I encourage you to share this in the Website Meeting tomorrow at 6:30 PM IST. Share your screen, and walk us through it. It's cool to take and share feedback. Check the [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit) here to include this PR.

You're welcome at the meeting

Check to [community calendar](https://layer5.io/community/calendar) for more details of meetings at Layers.

**Comment by leecalcote** [comment]
Closing in lieu of #9952

File: docs/pages/guides/configuration-management/configuration-management.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/guides/configuration-management/configuration-management.md]: @neerajmanivarnan you didn't incorporate my feedback into your new PR.

**Pull request closed without merging**